### ISSUE TITLE
Acid fast bacteria sputum culture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10974]
**Preferred term label:**
Acid fast bacteria sputum culture

**Synonyms**
Mycobacterium tuberculosis; AFB

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:6000963

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tuberculosis
Pulmonary aspergillosis (see https://pubmed.ncbi.nlm.nih.gov/39678851/)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001328
- label: Positive Mycobacterium tuberculosis sputum culture
- parents: [Sputum infectious agent (HP:0430107)](https://hpo.jax.org/app/browse/term/HP:0430107)
- pmid: 
- def: Growth of mycobacterium tuberculosis in a sputum culture.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10974
- synonyms: Acid fast bacteria sputum culture(NONE)
